get_github_coderabbit_review/process_pr_comments.py

```
@@ -0,0 +1,243 @@
+            'body_length': len(review.get('body', ''))
+        }
+        structured_data['review_comments'].append(structured_review)
+
+    # レビューコメント由来のCodeRabbit件数も加算
+    coderabbit_count += sum(
+        1
+        for r in reviews
+        if 'coderabbit' in r.get('user', {}).get('login', '').lower()
+    )
+
+    # アクショナブルアイテムを抽出（CodeRabbitコメント中心）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)